omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/RetrospectiveVisitFragmentControllerTest.java

```diff
@@ -165,7 +166,48 @@ public void test_shouldSetEndDateToCurrentTimeIfEndDateIsCurrentDay() throws Exc
 
     }
 
+    @Test
+    public void test_shouldCreateNewRetrospectiveVisitWithTimezoneAndDontChangeEndDateToEndOfDay() throws Exception {
```

Comment:
These two tests pass even without your above changes. Was your intention to start with failing tests and then make changes which make them pass? In other words, are these tests to cater for the changes in this pull request?

Reply:
Hello @dkayiwa those 2 tests on my side, only pass with my new changes.

Reply:
@icrc-jfigueiredo get a copy of the module without your changes. Then put in only those two tests. You will notices that the both pass on compiling.